Makefile

```diff
@@ -333,6 +335,9 @@ build-windows: .prebuild prepare-enterprise
 
 build: build-linux build-windows build-osx
 
+build-current: .prebuild prepare-enterprise
```

Comment:
`dist` does it for all operating systems which I was wanting to avoid because people were just running `make` and getting errors because they didn't have cross-compilation set up or the zip library installed